src/transformers/training_args.py

```diff
@@ -1213,6 +1213,15 @@ class TrainingArguments:
         },
     )
 
+    split_batches: Optional[bool] = field(
```

Comment:
It would be better if we could add usage intructions for this training argument in the docstring

Reply:
Done, good call!